configure.py

```diff
@@ -1906,8 +1906,7 @@ def query_seastar_flags(pc_file, link_static_cxx=False):
                 f.write('   libs = {}\n'.format(local_libs))
             else:
                 f.write('build $builddir/{}/{}: {}.{} {} | {}\n'.format(mode, binary, regular_link_rule, mode, str.join(' ', objs), seastar_dep))
-                if has_thrift:
-                    f.write('   libs =  {} {} $seastar_libs_{} $libs\n'.format(thrift_libs, maybe_static(args.staticboost, '-lboost_system'), mode))
+                f.write('   libs = {}\n'.format(local_libs))
```

Comment:
Seems like this fixes a bug as well: we didn't pass any libs if `has_thrift` was `False`.